doc/guide/admin.rst

```diff
@@ -852,6 +852,272 @@ to be added to one or more PAM stacks, e.g.
 
+********************
+Pre-flight Checklist
+********************
+
+Here are some things to check before going live with a new Flux system
+instance.
+
+Do I have all the right packages installed?
+===========================================
+
+.. list-table::
+   :header-rows: 1
+
+   * - Package name
+     - Where
+
+   * - flux-core
+     - All nodes
+
+   * - flux-security
+     - All nodes
+
+   * - flux-sched
+     - All nodes
+
+   * - flux-pam (optional)
+     - All nodes
+
+   * - flux-accounting (optional)
+     - Leader (management) node only
+
+Does /var/lib/flux have plenty of space on the leader node?
+===========================================================
+
+Flux stores its databases and KVS dumps in the ``statedir`` on the
+leader (management) node.  Storage consumption depend on usage, the
+size of the cluster, and other factors but be generous as running out
+of space on the leader node is catastrophic to Flux.
+
+The ``statedir`` is created automatically by systemd if it does not
+exist when Flux is started.  If you are creating it, it should be owned
+by the ``flux`` user and private to that user.
+
+Is Munge working?
+=================
+
+Munge daemons must be running on every node, clocks must be synchronized,
+and all nodes should be using the same pre-shared munge key.
+
+.. code-block:: console
+
+  $ pdsh -a systemctl is-active munge | dshbak -c
+  ----------------
+  test[0-7]
+  ----------------
+  active
+
+  $ pdsh -a "timedatectl | grep synchronized:" | dshbak -c
+  ----------------
+  test[0-7]
+  ----------------
+  System clock synchronized: yes
+
+  # spot check
+  $ echo xyz | ssh test1 munge | ssh test2 unmunge
+  STATUS:          Success (0)
+  ENCODE_HOST:     test1 (192.168.88.246)
+  ENCODE_TIME:     2024-04-18 09:41:21 -0700 (1713458481)
+  DECODE_TIME:     2024-04-18 09:41:21 -0700 (1713458481)
+  TTL:             300
+  CIPHER:          aes128 (4)
+  MAC:             sha256 (5)
+  ZIP:             none (0)
+  UID:             testuser (100)
+  GID:             testuser (100)
+  LENGTH:          4
+
+Are users set up properly?
+==========================
+
+Flux requires that the ``flux`` user and all other users that will be
+using Flux have the a consistent UID assignment across the cluster.
+
+.. code-block:: console
+
+  $ pdsh -a id flux | dshbak -c
+  ----------------
+  test[0-7]
+  ----------------
+  uid=500(flux) gid=500(flux) groups=500(flux)
+
+Is the Flux network certificate synced?
+=======================================
+
+The network certificate should be identical on all nodes and should
+only be readable by the ``flux`` user:
+
+.. code-block:: console
+
+  $ sudo pdsh -a md5sum /etc/flux/system/curve.cert | dshbak -c
+  ----------------
+  test[0-7]
+  ----------------
+  1b3c226159b9041d357a924841845cec  /etc/flux/system/curve.cert
+
+  $ pdsh -a stat -c '"%U %A"' /etc/flux/system/curve.cert | dshbak -c
+  ----------------
+  test[0-7]
+  ----------------
+  flux -r--------
```

Comment:
add link to the section on cert creation?

Reply:
I didn't link back to the relevant sections on all the other checklist items - do you think that would be a good idea?

Reply:
I don't think it's necessary for all of them.  This one stood out to me as a "did I completely forget a step?"  vs. other ones.  Like starting a stopped queue, your simple example is more than enough information to move forward.